lib/utils.js

```diff
@@ -28,9 +30,26 @@ const getTestReporter = function getTestReporter (opts) {
   return isForceLogMode ? 'spec' : (process.env.REPORTER ? process.env.REPORTER : opts.testReporter);
 };
 
+const pExec = function pExec (cmd, args, opts = {}) {
+  return new B(function (resolve, reject) {
+    exec(`${cmd} ${args.join(' ')}`, opts, function (err, stdout, stderr) { // eslint-disable-line promise/prefer-await-to-callbacks
```

Comment:
I'd rather use specialized module to join shell arguments, otherwise this will fail if any of the arguments contains spaces

Reply:
also, the original teen_process uses `spawn` if this makes any difference

Reply:
Sure.

`spawn` just gives better control over stdio, which is not necessary here.